aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -248,7 +248,15 @@ def calcjob_cleanworkdir(calcjobs, past_days, older_than, computers, force):
             echo.echo_critical('if no explicit calcjobs are specified, at least one filtering option is required')
 
     calcjobs_pks = [calcjob.pk for calcjob in calcjobs]
-    path_mapping = get_calcjob_remote_paths(calcjobs_pks, past_days, older_than, computers)
+    path_mapping = get_calcjob_remote_paths(
+        calcjobs_pks,
+        past_days,
+        older_than,
+        computers,
+        exit_status=exit_status,
+        include_calcjob_id=False,
```

Comment:
You are adding this, but I don't think you actually use the `calcjob_id`? If it is not necessary, I would not add this argument now.

Reply:
right yes, since now it just owrk with RemoteData. I have just removed it.